.pre-commit-hooks.yaml

```diff
@@ -40,8 +40,10 @@
 -   id: golangci-lint
     name: 'golangci-lint'
     entry: run-golangci-lint.sh
-    files: '\.go$'
+    types: [go]
     language: 'script'
+    args: ['run', '--skip-dirs', 'vendor', '--fast', '--fix', '--new']
```

Comment:
Looking at the golangci-lint docs, it sounds like `vendor` is always skipped.

For the other flags, I'm not sure we should change the default behaviour. When people update their version of this hook they wont get the same behaviour.

I was thinking that you could specify these flags in your config, now that the script uses `"$@"`